packages/paste-core/components/progress-steps/src/ProgressStepContent.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, type BoxProps } from "@twilio-paste/box";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import React from "react";
+
+export interface ProgressStepContentProps extends HTMLPasteProps<"div"> {
+  children?: React.ReactNode;
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   *
+   * @default 'PROGRESS_STEP_CONTENT'
+   * @type {BoxProps['element']}
+   * @memberof ProgressStepContentProps
+   */
+  element?: BoxProps["element"];
+}
+
+export const ProgressStepContent = React.forwardRef<HTMLDivElement, ProgressStepContentProps>(
+  ({ element = "PROGRESS_STEP_CONTENT", ...props }, ref) => {
+    return (
+      <Box element={element} color="colorText" ref={ref} paddingY="space40">
```

Comment:
suggestion: I think ProgressStepContent should have a paddingY prop, with a default value of space30. Different content will need different amounts of vertical padding (like in the doc site example), and then we can help the consumer not have to wrap things in a bunch of Boxes.

Reply:
Could we use space40? just so it gives a little extra space so its very clear what the label is vs. the content. Would this spacing be applied to both the top and bottom?

Reply:
used `space40`